### ISSUE TITLE
PHPUnit will not run the test on Windows PC using Better PHPUnit:run

### ISSUE THREAD

**Issue by Pen-y-Fan** [calebporzio/better-phpunit#103]
## What are the steps to reproduce this issue?

1. Fresh install of PHPUnit 
2. Run a test using Better PHP Unit:run

## What happens?

Command windows will flash up and close

VSCode terminal will open and terminate without running the test.

```sh
> Executing task: c:/laragon/www/GildedRose/GildedRose-Refactoring-Kata-php7/php7/vendor/bin/phpunit c:/laragon/www/GildedRose/GildedRose-Refactoring-Kata-php7/php7/test/GildedRoseTest.php --filter '^.*::gilded_rose' <


Terminal will be reused by tasks, press any key to close it.
```

## What were you expecting to happen?

The test should run.

If I add .bat to the phpunit.bat command and manually run the test it will run

```sh
PS C:\laragon\www\GildedRose\GildedRose-Refactoring-Kata-php7\php7> c:/laragon/www/GildedRose/GildedRose-Refactoring-Kata-php7/php7/vendor/bin/phpunit.bat c:/laragon/www/GildedRose/GildedRose-Refactoring-Kata-php7/php7/test/GildedRoseTest.php --filter '^.*::gilded_rose'
PHPUnit 8.1.3 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.3.8
Configuration: C:\laragon\www\GildedRose\GildedRose-Refactoring-Kata-php7\php7\phpunit.xml

...............................................................  63 / 173 ( 36%)
............................................................... 126 / 173 ( 72%)
...............................................                 173 / 173 (100%)

Time: 150 ms, Memory: 4.00 MB

OK (173 tests, 173 assertions)
```

Also if I rename vendor\bin\phpunit (Linux/Mac) file to any other name and re-run the test it will run.

```sh
> Executing task: c:/laragon/www/GildedRose/GildedRose-Refactoring-Kata-php7/php7/vendor/bin/phpunit c:/laragon/www/GildedRose/GildedRose-Refactoring-Kata-php7/php7/test/GildedRoseTest.php --filter '^.*::gilded_rose' <

PHPUnit 8.1.3 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.3.8
Configuration: C:\laragon\www\GildedRose\GildedRose-Refactoring-Kata-php7\php7\phpunit.xml

...............................................................  63 / 173 ( 36%)
............................................................... 126 / 173 ( 72%)
...............................................                 173 / 173 (100%)

Time: 91 ms, Memory: 4.00 MB

OK (173 tests, 173 assertions)

Terminal will be reused by tasks, press any key to close it.
```

## Any logs, error output, etc?

None, a window does flash open, I suspect with is windows trying to run the Linux/Mac files as though it was a batch file.

## Any other comments?

The temp solution is to rename PHPUnit, the permanent solution is to update the command to add .bat on a windows PC.

As you can see from the above the $ has already been removed on my PC, I'll do another update with:

```js
    get windowsSuffix() {
        return process.platform === "win32"
            ? '.bat'
            : '';
    }
// ... in     get binary() {
        return this.subDirectory
            ? this._normalizePath(path.join(this.subDirectory, 'vendor', 'bin', 'phpunit'+this.windowsSuffix))
            : this._normalizePath(path.join(vscode.workspace.rootPath, 'vendor', 'bin', 'phpunit'+this.windowsSuffix));
    }
```

I've tested today and it been working fine on my Windows PC. (as you can see from above it also has the $ removed from the filter) #100 / #102 

## What versions are you using?
**Operating System:** Windows 7 and 10
**Package Version:** 2.5

I'll submit a separate request, based on #100, so they can be approved together if required.

**Comment by keizah7** [comment]
Same issue

**Comment by Pen-y-Fan** [comment]
> Same issue @keizah7 

I have posted a fix, which is awaiting approval: see https://github.com/calebporzio/better-phpunit/blob/3e7d64fb65bfb4de9d4e92868e3e7ecd947358f5/src/phpunit-command.js

Please, can you:
1. Download this **phpunit-command.js** file from the link above? 
2. Close VS Code
3. Backup the **phpunit-command.js** file in C:\Users\<yourusername>\.vscode\extensions\calebporzio.better-phpunit-1.5.0\src 
4. Replace it with the download file from 1. 
5. Reopen VS Code
6. Run some tests.

Please let us know how you get on. I hope if other Windows users confirm the fix works from them then the fix will be approved.

**Comment by keizah7** [comment]
> > Same issue @keizah7
> 
> I have posted a fix, which is awaiting approval: see https://github.com/calebporzio/better-phpunit/blob/3e7d64fb65bfb4de9d4e92868e3e7ecd947358f5/src/phpunit-command.js
> 
> Please, can you:
> 
> 1. Download this **phpunit-command.js** file from the link above?
> 2. Close VS Code
> 3. Backup the **phpunit-command.js** file in C:\Users<yourusername>.vscode\extensions\calebporzio.better-phpunit-1.5.0\src
> 4. Replace it with the download file from 1.
> 5. Reopen VS Code
> 6. Run some tests.
> 
> Please let us know how you get on. I hope if other Windows users confirm the fix works from them then the fix will be approved.

I did steps, like you said and problem is fixed. I am using W10 Pro

**Comment by calebporzio** [comment]
Should be fixed in v1.5.1. Re-open if it's not. Thanks.
